associated-press/cdk/bin/cdk.ts

```diff
@@ -0,0 +1,12 @@
+import 'source-map-support/register';
+import { App } from 'aws-cdk-lib';
+import { AssociatedPressFeed } from '../lib/associated-press-feed';
+import { InstanceSize } from 'aws-cdk-lib/aws-ec2';
+
+const app = new App();
+new AssociatedPressFeed(app, 'AssociatedPressFeed-CODE', {
+	stack: 'media-service',
+	stage: 'CODE',
+	instanceSize: InstanceSize.MICRO,
+	domainName: 'code-guardian.com'
```

Comment:
We don't own this domain name, so will not be able to create a cert for it.  We should either change it to an appropriate Guardian owned domain or (ideally) get rid of it since this app doesn't need to be accessed via http